src/elm/Session/LoggedIn.elm

```diff
@@ -160,7 +160,7 @@ initModel shared authModel accountName selectedCommunity =
     , auth = authModel
     , feedback = Hidden
     , showCommunitySelector = False
-    , hasShop = False
```

Comment:
I've added this from the other PR #330 so I could do a deploy for MUDA (no one is being able to claim actions!!)